docs/application/index.md

```diff
@@ -1,178 +1,323 @@
+	transition-duration: 0.3s;
+}
 
-![Tizen architecture](media/what_is_tizen_architecture.png)
+</style>
 
-The Tizen platform also allows you to develop a hybrid application package where native and Web applications are packaged together to make more powerful applications. The Tizen platform ensures that all Tizen applications have consistent look and feel, regardless of whether you use the native or Web framework to create them.
+<section id ="main">
+Tizen is a user-interactive and service-oriented open source project that allows you to create feature-rich applications for multiple device categories.
 
+To get started with the development of your own Tizen applications:
 
+<div class="docs-ui-get-started">
+	<div class="docs-content-boxes content-boxes-1 content-boxes-icon-with-title docs-content-boxes-brand-boxes" data-animationoffset="100%" style="margin-top:10px;margin-bottom:10px;">
+		<div class="column docs-column content-box-column content-box-column-1 content-box-hover content-box-column-first-in-row">
+			<div class="col content-box-wrapper content-wrapper-background link-type-text icon-wrapper-hover-animation-pulsate" data-animationoffset="100%" style="background-color:#ffffff;">
+				<div class="vscom-boxes-body">
+					<p align="center"><img alt="" x-hidden-focus" src="./media/icon_get_started_wearable.png" height="90"></p>
+					<p class="vscom-boxes-text" align="center">
+					<a href="get-started/wearable.md" target="_self">Get Started for Wearable</a></p>
+				</div>
+			</div>
+		</div>
+		<div class="column docs-column content-box-column content-box-column-1 content-box-hover">
+			<div class="col content-box-wrapper content-wrapper-background link-type-text icon-wrapper-hover-animation-pulsate" data-animationoffset="100%" style="background-color:#ffffff;">
+				<div class="vscom-boxes-body">
+					<p align="center"><img alt="" x-hidden-focus" src="./media/icon_get_started_tv.png" height="90"></p>
+					<p class="vscom-boxes-text" align="center">
+					<a href="get-started/tv.md" target="_self">Get Started for TV</a></p>
+				</div>
+			</div>
+		</div>
+		<div class="column docs-column content-box-column content-box-hover content-box-column-last content-box-column-last-in-row">
+			<div class="col content-box-wrapper content-wrapper-background link-type-text icon-wrapper-hover-animation-pulsate" data-animationoffset="100%" style="background-color:#ffffff;">
+				<div class="vscom-boxes-body">
+					<p align="center"><img alt="" x-hidden-focus" src="./media/icon_get_started_mobile.png" height="90"></p>
+					<p class="vscom-boxes-text" align="center">
+					<a href="get-started/mobile.md" target="_self">Get Started for Mobile</a></p>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
 
-To become familiar with some terms that you encounter throughout this site, see [Glossary](../glossary.md).
+## Tizen Extensions for Visual Studio Family
+
+<div class="docs-ui-get-started">
```

Comment:
For the boxes under Tizen Extensions for Visual Studio Family,
Could we remove the IDE screenshots? It is making the cards look clustered. 
For example:

-------
"icon"
Visual Studio Tools for Tizen
Develop Tizen .NET.....

Learn More

--------

Please let me know your views. Kindly ignore if invalid.